shark/shark_downloader.py

```diff
@@ -110,7 +110,9 @@ def gs_download_model():
             np.load(os.path.join(model_dir, "upstream_hash.npy"))
         )
         if local_hash != upstream_hash:
-            gs_download_model()
```

Comment:
how does model downloading occur if not when the hash is nonmatching?

Reply:
If ~/.local/shark_tank/ is deleted, as directed by the print statement in check_dir_exists

Reply:
can we add a flag that disables the hash auto download and leave it enabled by default?